.env

```diff
@@ -0,0 +1 @@
+BRANCH_NAME='master'
```

Comment:
why we need this file?

Reply:
I've completed according to revit-model-browser. Branch name is set in external file there.
I'll avoid using of this file for d4r and set branch name in Jenkinsfile directly.